corgidrp/detector.py

```diff
@@ -4,6 +4,14 @@
 import corgidrp.data as data
 from scipy import interpolate
 from scipy.ndimage import median_filter
+import glob
+import os
+import astropy.io.fits as fits
+import photutils.centroids as centr
+from pathlib import Path
+from scipy.ndimage import gaussian_filter as gauss
+from photutils.aperture import CircularAperture
+from scipy import ndimage
 
 def create_flatfield(flat_dataset):
```

Comment:
This was a placeholder function we were using for some other code (in particular, the bad pixel map). We should eventually get rid of it if we're not using it, but this can be for a future PR.